docs/getting-started/getting-started.md

```diff
@@ -196,7 +196,7 @@ training_args = {
     'dry_run': False,  
 }
 
-from fedbiomed.researcher.experiment import Experiment
+from fedbiomed.researcher.federated_workflows import Experiment
```

Comment:
Nice pick @ybouilla 

But there are many other hanging `fedbiomed.researcher.experiment` (in docs, notebooks, docstrings)
Don't we want to remove them all in that branch ?

Reply:
To gain us time, I pushed a proposed commit to remove these occurrences. Is it ok for you ?